presto-docs/src/main/sphinx/security/server.rst

```diff
@@ -0,0 +1,254 @@
+
+``SystemAccessControl`` implementations have two responsibilities:
+ * Verifying whether or not a given Kerberos principal is authorized to execute
+   queries as a specific user.
+ * Determining whether or not a given user can alter values for a given system
+   property.
+
+The implementation of ``SystemAccessControl`` and
+``SystemAccessControlFactory`` must be wrapped as a plugin and installed on the
+Presto cluster.
+
+Presto Coordinator Node Configuration
+-------------------------------------
+
+config.properties
+^^^^^^^^^^^^^^^^^
+
+Kerberos authentication is configured in the coordinator node's
+config.properties file. The entries that need to be added are listed below.
+
+.. code-block:: none
+
+  http.server.authentication.enabled=true
+
+  http.server.authentication.krb5.service-name=presto
+  http.server.authentication.krb5.keytab=/etc/presto/presto.keytab
+  http.authentication.krb5.config=/etc/krb5.conf
+
+  http-server.https.enabled=true
+  http-server.https.port=7778
+
+  http-server.https.keystore.path=/etc/presto_keystore.jks
+  http-server.https.keystore.key=keystore_password
+
+* ``http.server.authentication.enabled``: Defines that the Presto coordinator should
+  enable authentication. Must be set to ``true``.
+* ``http.server.authentication.krb5.service-name``: Defines the Kerberos server
+  name for the Presto coordinator. Must match the Kerberos principal.
+* ``http.server.authentication.krb5.keytab``: The location of the keytab that
+  can be used to authenticate the Kerberos principal specified in
+  ``http.server.authentication.krb5.service-name``.
+* ``http.authentication.krb5.config``: The location of the Kerberos
+  configuration file.
+* ``http-server.https.enabled``: Specifies if Presto coordinator should listen
+  for HTTPS.
+* ``http-server.https.port``: HTTPS server port.
+* ``http-server.https.keystore.path``: The location of the Java Keystore file
+  that will be used to secure TLS.
+* ``http-server.https.keystore.key``: The password for the keystore. This must
+  match the password you specified when creating the keystore.
+
+access-controls.properties
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once a plugin that implements ``SystemAccessControl`` and
+``SystemAccessControlFactory`` has been installed, it is configured using an
+access-controls.properties file.
+
+.. code-block:: none
+
+  access-control.name=custom-access-control
+  custom-property1=custom-value1
+  custom-property2=custom-value2
+  ...
+
+The ``access-control.name`` property is used by Presto to find a registered
+``SystemAccessControlFactory``. The remaining properties are passed as a map to
+``SystemAccessControlFactory.create(Map<String, String> config)``.
+
+.. _coordinator-troubleshooting:
+
+Troubleshooting
+---------------
+
+Getting Kerberos authentication can be challenging. You can independently
+verify some of the configuration outside of Presto to help narrow your focus
+when trying to solve a problem.
+
+Kerberos
```

Comment:
Technically, a successful `kinit` verifies that the KDC knows who the principal is and that the keytab file is valid. Got a suggestion for a section title that isn't overly verbose ;-)